### ISSUE TITLE
Add counter for number of times eligibility screener has been used

### ISSUE THREAD

**Issue by kadrianne** [codefordenver/expunge-colorado-screener#37]
Increase by one on submission of eligibility survey. Maybe easier by counting number of records stored? Blocked by https://github.com/codefordenver/expunge-colorado-screener/issues/33

**Comment by eemshi** [comment]
I agree the count should come from the data store; maybe this should be labeled blocked until that is ready to consume

**Comment by eemshi** [comment]
Should we figure out a way to distinguish usage from testing and real usage? Perhaps a staging site and a prod site that are connected to different dbs?
